utils/src/test/kotlin/ArchiveUtilsTest.kt

```diff
@@ -30,76 +30,132 @@ import java.io.File
 
 import org.ossreviewtoolkit.utils.test.createTestTempDir
 
-class ArchiveUtilsTest : StringSpec() {
+class ArchiveUtilsTest : WordSpec() {
     private lateinit var outputDir: File
 
     override fun beforeTest(testCase: TestCase) {
         outputDir = createTestTempDir()
     }
 
     init {
-        "Tar GZ archive can be unpacked" {
-            val archive = File("src/test/assets/test.tar.gz")
+        "Tar archives" should {
```

Comment:
Consider grouping it by function, like:

"unpack()" should { "unpack Tar archvies"

Reply:
The current grouping was suggested by @sschuberth. Could you two come to an agreement, please?

Reply:
With the "Consider" prefix a wanted to indicate that it's fully optional and only would like it to be considered, but leave the decision up to you. So no agreement with me needed. (Wasn't aware of @sschuberth 's suggestion)